layers/+tags/gtags/packages.el

```diff
@@ -14,6 +14,7 @@
   '(
     ggtags
     (helm-gtags :requires helm)
+    (counsel-gtags :toggle (configuration-layer/layer-usedp 'ivy))
```

Comment:
Please use the new `:requires` keyword for this sort of thing. (See line above.)